### ISSUE TITLE
Windows: Won on my pc but now the UI is stuck here post catch up animation 

### ISSUE THREAD

**Issue by metalaureate** [tari-project/universe#1198]
**Describe the bug**
![image](https://github.com/user-attachments/assets/1ba8f811-ad80-45df-99ec-9404f86e8b4e)
in this state, replay doesnt work
and there is no way to get it back to the original block animation
however when miimized it will report on additional winnings, just without the animation

**Comment by Misieq01** [comment]
Got the same problem on `0.8.13` dumping logs for debugging
![image](https://github.com/user-attachments/assets/2e2cbba1-4bd9-43ea-b20d-605c31e3fdd4)
[logs.zip](https://github.com/user-attachments/files/18110896/logs.zip)

**Comment by shanimal08** [comment]
yep definitely related to the `Error getting network hash rate and block reward:` error & commands caching with `Already getting [tx/metrics]`


![image](https://github.com/user-attachments/assets/47730720-d5da-4fe2-b002-88f8d1504622)
